config.default.yaml

```diff
@@ -145,16 +145,16 @@ electricity:
     H2: 168
 
   extendable_carriers:
-    Generator: [solar, onwind, offwind-ac, offwind-dc, OCGT]
+    Generator: [solar, onwind, offwind-ac, offwind-dc, OCGT, csp]
```

Comment:
Probably, it could make sense to move `csp` in the list to have all the renewable carriers grouped together.